internal/operands/common-instancetypes/kustomize.go

```diff
@@ -0,0 +1,164 @@
+package common_instancetypes
```

Comment:
Maybe rename this file to `mockresmap.go` to make it more obvious this is only test code?